src/foolscap/banana.py

```diff
@@ -121,10 +121,10 @@ def populateVocabTable(self, vocabStrings):
         contents of this table.
         """
 
-        out_vocabDict = dict(zip(vocabStrings, range(len(vocabStrings))))
+        out_vocabDict = dict(list(zip(vocabStrings, list(range(len(vocabStrings))))))
         self.outgoingVocabTableWasReplaced(out_vocabDict)
 
-        in_vocabDict = dict(zip(range(len(vocabStrings)), vocabStrings))
+        in_vocabDict = dict(list(zip(list(range(len(vocabStrings))), vocabStrings)))
```

Comment:
hm, I think this could just be `dict(enumerate(vocabStrings))`. I probably didn't know about `enumerate` back then, or maybe it didn't exist yet.

The `out_vocabDict` can probably be a simple transform of `enumerate(vocabStrings)`, but I'm not sure what.

Reply:
Changes auto-created by 2to3, will take a closer look.

Reply:
Fixed.